src/arrays/Arrays.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace arrays;
+
+class Arrays implements ArraysInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $new_array = [];
+        if (count($input) != 0) {
+            foreach ($input as $currentElement) {
+                for ($i = $currentElement; $i > 0; $i--) {
+                    $new_array[] = $currentElement;
+                }
+            }
+        }
+
+        return $new_array;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getUniqueValue(array $input): int
+    {
+        $uniqueArray = array_unique($input, SORT_NUMERIC);   //getting single values
+        $count_values = array_count_values($input);
+
+        foreach ($uniqueArray as $value) {                        //converting single values array into unique values
+            if ($count_values[$value] > 1) {
+                $key = array_search($value, $uniqueArray);
+                unset($uniqueArray[$key]);
```

Comment:
it is not good to unset array that you are looping

Reply:
> it is not good to unset array that you are looping

All changes done. Now I'm proud of my code here ;)